Homework_10/improved_calculator.py

```diff
@@ -0,0 +1,37 @@
+def addition(number_1, number_2, operataion):
+    if operataion == '+':
+        result = number_1 + number_2
+        return print(result)
+
+
+def deduction(number_1, number_2, operataion):
+    if operataion == '-':
+        result = number_1 - number_2
+        return print(result)
```

Comment:
the same situation for all functions

Reply:
![image](https://github.com/vithaiduchyk/hillel_python_basic/assets/51053483/24913dc1-34e7-47d0-ae6f-40f1cc8c7899)

Reply:
fixed